Realm/RLMSyncUtil_Private.h

```diff
@@ -62,6 +62,11 @@ extern NSString *const kRLMSyncRegisterKey;
 extern NSString *const kRLMSyncUnderlyingErrorKey;
 extern NSString *const kRLMSyncUserIDKey;
 
+extern NSString *const kRLMSyncRequestAuthorizationHeaderNameKey;
+extern NSString *const kRLMSyncRequestExtraHeadersKey;
+
+NSDictionary *RLMSyncRequestOptions(void);
```

Comment:
This should be a method on RLMSyncManager and should return a typed object rather than a dictionary with some magic keys.